.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] According to the project requirements, there is supposed to be a `test.md` file which should be ignored by git. I have posted a quote for that requirement below. Please make sure to fulfill all of the requirements of the project so your project can be approved.

     > A test.md file with the text "This file should be ignored by git".
     > Make sure that the last file is actually ignored by git and is not present in your GitHub repository.